packages/app/src/EthereumProviders.tsx

```diff
@@ -5,10 +5,27 @@ import { chain, configureChains, createClient, WagmiConfig } from "wagmi";
 import { alchemyProvider } from "wagmi/providers/alchemy";
 import { publicProvider } from "wagmi/providers/public";
 
-export const targetChainId = parseInt(process.env.CHAIN_ID || "0") || 5;
+export const targetChainId =
+  parseInt(process.env.NEXT_PUBLIC_CHAIN_ID || "0") || 5;
+const targetChains = [chain.mainnet];
+switch (targetChainId) {
+  case 3:
+    targetChains.push(chain.ropsten);
+    break;
+  case 4:
+    targetChains.push(chain.rinkeby);
+    break;
+  case 42:
+    targetChains.push(chain.kovan);
+    break;
+  case 5:
+  default:
+    targetChains.push(chain.goerli);
+    break;
+}
 
 export const { chains, provider, webSocketProvider } = configureChains(
-  [chain.mainnet, chain.goerli],
```

Comment:
The only reason I had this specified is that so RainbowKit would know which chains to tell you to switch to when you're on a different network. Ideally we'd just use the default chain list or all chains.